builtin/providers/triton/resource_vlan.go

```diff
@@ -0,0 +1,132 @@
+package triton
+
+import (
+	"errors"
+	"strconv"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/joyent/gosdc/cloudapi"
+)
+
+func resourceVLAN() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceVLANCreate,
+		Exists: resourceVLANExists,
+		Read:   resourceVLANRead,
+		Update: resourceVLANUpdate,
+		Delete: resourceVLANDelete,
+
+		Schema: map[string]*schema.Schema{
+			"vlan_id": {
+				Description: "number between 2-4095 indicating VLAN ID",
+				Required:    true,
+				ForceNew:    true,
+				Type:        schema.TypeInt,
+				ValidateFunc: func(val interface{}, field string) (warn []string, err []error) {
+					value := val.(int)
+					if value < 0 || value > 4095 {
```

Comment:
The description here doesn't match the condition - `2 <= value <= 4095` vs `0 < value <= 4095`?